src/oxli/kmer_hash.cc

```diff
@@ -49,6 +49,11 @@ Contact: khmer-project@idyll.org
 
 using namespace std;
 
+#define tbl \
+  "                                                                "\
+  /*ABCDEFGHIJKLMNOPQRSTUVWXYZ      abcdefghijklmnopqrstuvwxyz    */\
+  " TVGH  CD  M KN   YSAABW R       TVGH  CD  M KN   YSAABW R"
```

Comment:
Is the leading whitespace intentional?

Reply:
yes, you want to 'pad' the 'array' so that `tbl[ord("A")]` gives you the translation